go/app/main.go

```diff
@@ -83,20 +81,30 @@ func saveItem(name, category string) error {
 }
 
 func getItems(c echo.Context) error {
-	data, err := os.ReadFile(ItemsPath)
+	items, err := readItems(ItemsPath)
 	if err != nil {
-		errRes := Response{Message: "error while reading file"}
-		return c.JSON(http.StatusInternalServerError, errRes)
+		err := fmt.Errorf("error while reading or unmarshaling file: %w", err)
+		return err
 	}
 
-	var items []Item
-	err = json.Unmarshal(data, &items)
+	return c.JSON(http.StatusOK, items)
+}
+
+func readItems(filepath string) (Items, error) {
+	var items Items
+
+	data, err := os.ReadFile(filepath)
```

Comment:
[nits]
非常に細かいですがキャメルですかね！👀
```suggestion
func readItems(filePath string) (Items, error) {
	var items Items

	data, err := os.ReadFile(filePath)
```

Reply:
ありがとうございます！ 修正しました！